src/v/cloud_storage_clients/configuration.cc

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
```
    cloud_storage_clients: lift config logic from ...
    
    ... client impl
```

that can probably fit on one line?

Reply:
It's longer than 50 chars if I squash it in one line. That's the max length I aim for. Should I use something different?

Reply:
i'm just saying: let's make the commit subjects stand on their own so that `git log --oneline` makes sense

Reply:
I think we never really defined this... I confess that I write "too long" commit messages sometimes, but also sometimes do that "I'm a long...\n...line" thing when I really can't fit the sentence in one line.

Feel like maybe we should define a limit but make it longer than 50 chars (because we eat so many chars with our component: prefix)

Reply:
i would be fine with whatever length doesn't wrap on github (i think this is around 72 characters) even though this is longer than the default in most editors.